### PR TITLE
G Suite: Deactivate Finish Setup notices for non-admin users

### PR DESCRIPTION
**Pull request by stephanethomas**
WIP.

### CODE REVIEW

File: client/my-sites/email/email-management/gsuite-user-item/index.jsx [hunk 1]
```diff
@@ -20,40 +20,67 @@ import './style.scss';
...
+	const renderMailbox = () => {
+		return (
+			<ExternalLink
+				icon
+				href={ getMailboxLink() }
+				onClick={ props.onClick }
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				{ translate( 'Mailbox', { context: 'Link pointing to Gmail' } ) }
```
Comment (a8ci18n) [inline client/my-sites/email/email-management/gsuite-user-item/index.jsx]: :information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

File: client/my-sites/email/email-management/gsuite-user-item/index.jsx [hunk 2]
```diff
@@ -20,40 +20,67 @@ import './style.scss';
...
 			<ExternalLink
 				icon
-				className="gsuite-user-item"
 				href={ getLoginLink() }
 				onClick={ props.onClick }
 				target="_blank"
 				rel="noopener noreferrer"
 			>
-				{ translate( 'Manage', { context: 'Login to G Suite Manage' } ) }
+				{ translate( 'Manage', { context: 'Link pointing to Google Admin Console' } ) }
```
Comment (a8ci18n) [inline client/my-sites/email/email-management/gsuite-user-item/index.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Manage', { context: 'Login to G Suite Manage'} )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/gsuite-finish-setup-notices)

File: client/my-sites/email/email-management/gsuite-user-item/index.jsx [hunk 3]
```diff
@@ -20,40 +20,67 @@ import './style.scss';
...
 	};
+
 	const onCloseClickHandler = () => {
 		setDialogVisible( false );
 	};
 
+	const getMailboxLink = () => {
+		const { email } = props.user;
+
+		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel&continue=https://mail.google.com/`;
```
Comment (stephanethomas) [inline client/my-sites/email/email-management/gsuite-user-item/index.jsx:36]: The url should be:

https://accounts.google.com/AccountChooser?service=mail&continue=https://mail.google.com/mail/.

File: client/my-sites/email/email-management/gsuite-user-item/index.jsx [hunk 4]
```diff
@@ -20,47 +23,91 @@ import './style.scss';
...
+		return (
+			<ExternalLink
+				icon
+				href={ getGoogleAdminUrl( user.email ) }
+				onClick={ onClick }
+				target="_blank"
+				rel="noopener noreferrer"
+				title={ translate( 'Go to Google Admin to manage your G Suite account' ) }
+			>
+				{ translate( 'Manage', { context: 'Link pointing to Google Admin' } ) }
```
Comment (a8ci18n) [inline client/my-sites/email/email-management/gsuite-user-item/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **4** times:
`translate( 'Manage', { context: 'link to page to manage sites'} )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/gsuite-finish-setup-notices)


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

File: client/lib/gsuite/new-users.ts [hunk 5]
```diff
@@ -98,7 +98,7 @@ const validEmailCharacterField = ( { value, error }: GSuiteNewUserField ): GSuit
 	error:
 		! error && ! /^[0-9a-z_'-](\.?[0-9a-z_'-])*$/i.test( value )
 			? translate(
-					'Only number, letters, dashes, underscores, apostrophes and periods are allowed.'
+					'Only numbers, letters, dashes, underscores, apostrophes and periods are allowed.'
```
Comment (a8ci18n) [inline client/lib/gsuite/new-users.ts]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Pull request closed without merging**